CondCore/SiStripPlugins/scripts/G2GainsValidator.py

```diff
@@ -33,21 +33,30 @@ def getCommandOutput(command):
 ##############################################
 def getFCSR():
 ##############################################
-    out = subprocess.check_output(["curl", "-k", "-s", "https://cmsweb.cern.ch/t0wmadatasvc/prod/firstconditionsaferun"])
+    # temporary solution accessing URL that is not requiring a certificate but it is
```

Comment:
@JanChyczynski this python script is used by the SiStrip DB contact to validate G2 gains payload extracted from the multi-run harvesting tag against the conditions used *at that moment* in prompt.
To clarify it is by **no means** critical in any aspect as the same job can be done equally well by hand, thus I believe there is no need for such hasty patch.
What is the long term solution here?

Reply:
Good to know that it's not critical, I'll focus on fixing CondCore/Utilities/python/tier0.py now.

I am working on the long-term solution right now and the idea is to use X509_USER_CERT and (optionally) X509_USER_KEY environment variables to specify the path(s) to the certificate and add it to the curl command by the --cert flag (and optionally --key). Then you need to have a certificate installed and the env variables pointing to them on the machine you run it.

Reply:
I've tried to come up with a fix for this script at https://github.com/cms-sw/cmssw/pull/45867